style/reading/links-to-physics.less

```diff
@@ -19,15 +20,11 @@
     text-transform: uppercase;
     letter-spacing: 2px;
     text-align: center;
-    background: @physics-background-secondary;
+    background: (@physics-background-secondary);
 
-    &:before {
-      content: "\2014  ";
-    }
+    &:before { content: "\2014  ";}
```

Comment:
RE 3: ok, just making sure it was noticed. Seems fine to do it in subsequent PR's